app/Domains/Users/Policies/UserPolicy.php

```diff
@@ -14,7 +14,7 @@ class UserPolicy
     /**
      * Determine whether the authenticated user can delete an other user or not.
      *
-     * @param  User $user The resource entity from the authenticated user.
+     * @param User $user The resource entity from the authenticated user.
```

Comment:
Hey Can u align the User class with the same start as the return type in the docblock.